### PR TITLE
Update Android recommendations

### PR DESCRIPTION
**Pull request by tommytran732**
Update android recommendations

I have not included the explanation for why verified boot is important here yet, it will be done in a PR after the browser section is cleaned up.

### CODE REVIEW

**Comment by dngray** [comment]
> I have not included the explanation for why verified boot is important here yet, it will be done in a PR after the browser section is cleaned up.

Make sure you open an issue, so we don't forget.

**Comment by tommytran732** [comment]
Issue made

File: _includes/legacy/sections/android-operating-systems.html [hunk 1]
```diff
@@ -1,49 +1,33 @@
...
-  github="https://github.com/LineageOS"
+  website="https://calyxos.org"
+  gitlab="https://gitlab.com/calyxos"
 %}
 
-<h4>Worth Mentioning</h4>
+{% include legacy/cardv2.html
+  title="DivestOS"
+  image="/assets/img/legacy_png/3rd-party/divestos.png"
+  description='<p>DivestOS is a soft-fork of <a href="https://lineageos.org/">LineageOS</a> with signed builds, automated kernel vulnerability (CVE) patcher, bundled <a href="https://www.f-droid.org/">F-Droid</a> and <a href="https://github.com/microg/UnifiedNlp">UnifedNPL</a>.</p>
```
Comment (SkewedZeppelin) [inline _includes/legacy/sections/android-operating-systems.html:25]:
```suggestion
  description='<p>DivestOS is a soft-fork of <a href="https://lineageos.org/">LineageOS</a> with signed builds, automated kernel vulnerability (CVE) patcher, bundled <a href="https://www.f-droid.org/">F-Droid</a> and <a href="https://github.com/microg/UnifiedNlp">UnifedNlp</a>.</p>
```

File: _includes/legacy/sections/android-operating-systems.html [hunk 2]
```diff
@@ -1,49 +1,33 @@
...
-    {% include badge.html
-    color="info"
-    text="Add-on Package"
-    %}
-    - A project that aims to reimplement the proprietary Google Play Services in the Android operating system with a FLOSS replacement. The microG project also maintains a fork of LineageOS with microG and F-Droid preinstalled at <a href="https://lineage.microg.org/">Lineage for microG</a>.
-  </li>
-</ul>
+  <p>Not all devices that it supports support <a href="https://source.android.com/security/verifiedboot">verified boot</a>, and DivestOS does not handle firmware updates well currently. For Pixel devices, we still recommend using GrapheneOS or CalyxOS. For other <a href="https://divestos.org/index.php?page=devices&base=LineageOS">supported devices</a>, DivestOS is good option.</p>'
+  badges="info:AOSP"
+  website="https://www.divestos.org/"
```
Comment (SkewedZeppelin) [inline _includes/legacy/sections/android-operating-systems.html:29]:
```suggestion
  website="https://www.divestos.org/"
  tor="divestoseb5nncsydt7zzf5hrfg44md4bxqjs5ifcv4t7gt7u6ohjyyd.onion"
```

File: _includes/legacy/sections/android-operating-systems.html [hunk 3]
```diff
@@ -1,49 +1,33 @@
...
-  <li>
-    <a href="https://microg.org/">MicroG</a>
-    {% include badge.html
-    color="info"
-    text="Add-on Package"
-    %}
-    - A project that aims to reimplement the proprietary Google Play Services in the Android operating system with a FLOSS replacement. The microG project also maintains a fork of LineageOS with microG and F-Droid preinstalled at <a href="https://lineage.microg.org/">Lineage for microG</a>.
-  </li>
-</ul>
+  <p>Not all devices that it supports support <a href="https://source.android.com/security/verifiedboot">verified boot</a>, and DivestOS does not handle firmware updates well currently. For Pixel devices, we still recommend using GrapheneOS or CalyxOS. For other <a href="https://divestos.org/index.php?page=devices&base=LineageOS">supported devices</a>, DivestOS is good option.</p>'
```
Comment (SkewedZeppelin) [inline _includes/legacy/sections/android-operating-systems.html:27]:
```suggestion
  <p>Not all of its supported devices support <a href="https://source.android.com/security/verifiedboot">verified boot</a>, and DivestOS does not handle firmware updates well currently. For Pixel devices, we still recommend using GrapheneOS or CalyxOS. For other <a href="https://divestos.org/index.php?page=devices&base=LineageOS">supported devices</a>, DivestOS is good option.</p>'
```

File: _includes/legacy/sections/android-operating-systems.html [hunk 4]
```diff
@@ -1,49 +1,33 @@
...
-  <li>
-    <a href="https://microg.org/">MicroG</a>
-    {% include badge.html
-    color="info"
-    text="Add-on Package"
-    %}
-    - A project that aims to reimplement the proprietary Google Play Services in the Android operating system with a FLOSS replacement. The microG project also maintains a fork of LineageOS with microG and F-Droid preinstalled at <a href="https://lineage.microg.org/">Lineage for microG</a>.
-  </li>
-</ul>
+  <p>Not all of its supported devices support <a href="https://source.android.com/security/verifiedboot">verified boot</a>, and DivestOS does not handle firmware updates well currently. For Pixel devices, we still recommend using GrapheneOS or CalyxOS. For other <a href="https://divestos.org/index.php?page=devices&base=LineageOS">supported devices</a>, DivestOS is good option.</p>'
```
Comment (SkewedZeppelin) [inline _includes/legacy/sections/android-operating-systems.html:27]:
```suggestion
  <p>Not all of its supported devices support <a href="https://source.android.com/security/verifiedboot">verified boot</a>, and DivestOS does not handle firmware updates well currently. For Pixel devices, we still recommend using GrapheneOS or CalyxOS. For other <a href="https://divestos.org/index.php?page=devices&base=LineageOS">supported devices</a>, DivestOS is a good option.</p>'
```

File: _includes/legacy/sections/android-operating-systems.html [hunk 5]
```diff
@@ -1,39 +1,33 @@
...
+  title="DivestOS"
+  image="/assets/img/legacy_png/3rd-party/divestos.png"
+  description='<p>DivestOS is a soft-fork of <a href="https://lineageos.org/">LineageOS</a> with signed builds, automated kernel vulnerability (CVE) patcher, bundled <a href="https://www.f-droid.org/">F-Droid</a> and <a href="https://github.com/microg/UnifiedNlp">UnifedNlp</a>.</p>
+  <p>Not all of its supported devices support <a href="https://source.android.com/security/verifiedboot">verified boot</a>, and DivestOS does not handle firmware updates well currently. For Pixel devices, we still recommend using GrapheneOS or CalyxOS. For other <a href="https://divestos.org/index.php?page=devices&base=LineageOS">supported devices</a>, DivestOS is a good option.</p>'
   badges="info:AOSP"
-  website="https://grapheneos.org/"
-  github="https://github.com/GrapheneOS/"
+  website="https://www.divestos.org/"
+  tor="divestoseb5nncsydt7zzf5hrfg44md4bxqjs5ifcv4t7gt7u6ohjyyd.onion"
```
Comment (SkewedZeppelin) [inline _includes/legacy/sections/android-operating-systems.html:29]: wasn't clickable, apologies
```suggestion
  tor="http://divestoseb5nncsydt7zzf5hrfg44md4bxqjs5ifcv4t7gt7u6ohjyyd.onion/"
```

File: _includes/legacy/sections/android-operating-systems.html [hunk 6]
```diff
@@ -1,39 +1,33 @@
...
 %}
 
 {% include legacy/cardv2.html
-  title="GrapheneOS"
-  image="/assets/img/legacy_svg/3rd-party/grapheneos.svg"
-  image-dark="/assets/img/legacy_svg/3rd-party/grapheneos-dark.svg"
-  description='GrapheneOS (formerly known as CopperheadOS) is a free and open-source security- and privacy-focused mobile operating system built on top of the Android Open Source Project. It specifically targets devices offering strong hardware security, currently only supporting Google Pixel phones.'
+  title="DivestOS"
+  image="/assets/img/legacy_png/3rd-party/divestos.png"
+  description='<p>DivestOS is a soft-fork of <a href="https://lineageos.org/">LineageOS</a> with signed builds, automated kernel vulnerability (CVE) patcher, bundled <a href="https://www.f-droid.org/">F-Droid</a> and <a href="https://github.com/microg/UnifiedNlp">UnifedNlp</a>.</p>
```
Comment (SkewedZeppelin) [inline _includes/legacy/sections/android-operating-systems.html:25]:
```suggestion
  description='<p>DivestOS is a soft-fork of <a href="https://lineageos.org/">LineageOS</a> with signed builds, automated kernel vulnerability (CVE) patching, fewer proprietary blobs, along with bundled <a href="https://www.f-droid.org/">F-Droid</a> and <a href="https://github.com/microg/UnifiedNlp">UnifedNlp</a>.</p>
```

**Comment by tommytran732** [comment]
@dngray Your <p> tag broke the PR, could you remove it

**Comment by dngray** [comment]
> @dngray Your
> 
> tag broke the PR, could you remove it

Fixed it.

File: _includes/legacy/sections/android-operating-systems.html (hunk 6 above)
Comment (dngray) [reply _includes/legacy/sections/android-operating-systems.html]: I've included these changes.

**Review by dngray — APPROVED** [review]
LGTM

**Review by SkewedZeppelin — APPROVED** [review]
Thank you @tommytran732 and @dngray!

File: _includes/legacy/sections/android-operating-systems.html [hunk 7]
```diff
@@ -1,39 +1,32 @@
...
+  title="DivestOS"
+  image="/assets/img/legacy_png/3rd-party/divestos.png"
+  description='DivestOS is a soft-fork of <a href="https://lineageos.org/">LineageOS</a> with signed builds, automated kernel vulnerability (CVE) patching, fewer proprietary blobs, along with bundled <a href="https://www.f-droid.org/">F-Droid</a> and <a href="https://github.com/microg/UnifiedNlp">UnifedNlp</a>. Not all of its supported devices support <a href="https://source.android.com/security/verifiedboot">verified boot</a>, and DivestOS does not handle firmware updates well currently. For Pixel devices, we still recommend using GrapheneOS or CalyxOS. For other <a href="https://divestos.org/index.php?page=devices&base=LineageOS">supported devices</a>, DivestOS is a good option.'
   badges="info:AOSP"
-  website="https://grapheneos.org/"
-  github="https://github.com/GrapheneOS/"
+  website="https://www.divestos.org/"
```
Comment (SkewedZeppelin) [inline _includes/legacy/sections/android-operating-systems.html:27]: there is no www.
```suggestion
  website="https://divestos.org/"
```

**Comment by Dyrimon** [comment]
@SkewedZeppelin www.divestos.org fails to connect to the server whereas divestos.org connects without any problems.

**Comment by SkewedZeppelin** [comment]
> @SkewedZeppelin [www.divestos.org](http://www.divestos.org) fails to connect to the server whereas divestos.org connects without any problems.

There was never a www.divestos.org.
I just updated DNS, httpd, and updated the certificate.

**Pull request merged**